docs/en/index.md

```diff
@@ -4,17 +4,17 @@ FreshRSS is an RSS aggregator and reader. It allows you to read and follow sever
 
 FreshRSS has a lot of features including:
 
-- RSS and Atom aggregation
-- Mark article as favorite if you liked it or if you want to read it later
-- Filter and search functionality helps to easily find articles
-- Statistics to show you the publishing frequency all the websites you follow
-- Import/export of your feeds into OPML format
-- Several themes created by the community
-- "Google Reader"-like API to connect Android applications
-- The application is "responsive," which means it adapts to small screens so you can bring articles in your pocket
-- Self-hosted: the code is free (under AGPL3 licence), so you can host your own instance of FreshRSS
-- Multi-user, so you can also host for your friends and family
-- And a lot more!
+* RSS and Atom aggregation
+* Mark article as favorite if you liked it or if you want to read it later
```

Comment:
My personal preference is for American English, but this is incongruent with the use of `favourite` and `brackets` elsewhere in the docs. In which direction shall I fix it up? Favorite/parentheses or favourite/brackets?

Reply:
I have the same preference :)